xml/System/StringNormalizationExtensions.xml

```diff
@@ -52,10 +52,13 @@
         <Parameter Name="value" Type="System.String" RefType="this" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="value">A string.</param>
+        <summary>Indicates whether the specified string is in Unicode normalization form C.</summary>
```

Comment:
Should "normalization form" be "normalized form"?

Reply:
I think using normalization form is ok.

---
In reply to: [141692770](https://github.com/dotnet/docs/pull/3260#discussion_r141692770) [](ancestors = 141692770)

Reply:
@BillWagner, normalization form is correct. A string is "normalized", but its "normalization form" is (whatever).